index.html

```diff
@@ -1,25 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/src/output.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/src/output.css" />
     <title>Baumeister</title>
-</head>
-<script>
-    const loadComponents = (id, url) =>  {
-        fetch(url)
+  </head>
+  <script>
+    const loadComponents = (id, url) => {
+      fetch(url)
         .then((response) => response.text())
         .then((html) => {
-            document.getElementById(id).innerHTML = html;
+          document.getElementById(id).innerHTML = html;
         });
-    }
+    };
 
     window.onload = () => {
-        loadComponents('hero', 'src/sections/Hero.html');    
-    }
-</script>
-<body>
+      loadComponents("hero", "src/sections/Hero.html");
```

Comment:
Here you are calling the Hero.html but I think you were supposed to call the footer section. In order to avoid such mistakes, when you are done with your tasks always check your deployment and see what's being rendered on the deployment link rather than the local host.